task/buildah/0.1/buildah.yaml

```diff
@@ -44,7 +44,11 @@ spec:
     type: string
   - default: "false"
     description: Determines if build will be executed without network access.
-    name: HERMETIC_BUILD
+    name: HERMETIC
+    type: string
+  - default: ""
+    description: Build dependencies that were previously prefetched.
+    name: PREFETCHED_CONTENT
```

Comment:
Just a nitpick for the name and description - does this reflect what the param actually is?

Reply:
I was also not satisfied with this. At first I wanted to somehow make this into a boolean value, but I couldn't find a reasonable way to do it. I tried to make this parameter dependent on a result emitted from the prefetch-dependencies task, but then the buildah task was be skipped altogether every time the prefetch-dependencies task was skipped :face_exhaling: 

I changed the parameter to actually reflect what it is, although I think it is slightly confusing when you look at the `buildah` task apart from the whole pipeline.

Reply:
> I tried to make this parameter dependent on a result emitted from the prefetch-dependencies task, but then the buildah task was be skipped altogether every time the prefetch-dependencies task was skipped :face_exhaling:

Oh, right :disappointed: 

The new name seems fine to me